### ISSUE TITLE
German language support?

### ISSUE THREAD

**Issue by einzelfabrik** [cmanon/wp-strava#63]
Is there a way to translate the values given by Strava ? On my german site it looks simply better!

**Comment by jrfoell** [comment]
Yes, thank you! If you go here: https://translate.wordpress.org/projects/wp-plugins/wp-strava/stable/de/default/ you can enter the translations and it will automatically create a translation file for the plugin that you (and others) can use.

**Comment by einzelfabrik** [comment]
No problem - I translated the missing German texts. German is now 100% translated. How long is this status "waiting"?

**Comment by jrfoell** [comment]
Hi @einzelfabrik - if you're willing, I'd like to add you as an Editor for the German language translations of the wp-strava plugin. Verbal confirmation of that here would help.

Having editor status granted is not a fast process from my experience, but once you're approved the updates should be easy. Thank you!

**Comment by einzelfabrik** [comment]
Thank you

**Comment by jrfoell** [comment]
I put in a request for you here, hopefully we hear back soon: https://make.wordpress.org/polyglots/2019/04/02/pte-request-for-wp-strava/#comment-290000
